openai.audio/Ballerina.toml

```diff
@@ -1,8 +1,13 @@
 [package]
+license = ["Apache-2.0"]
+keywords = ["AI", "Cost/Paid"]
```

Comment:
```suggestion
keywords = ["AI", "speech-to-text, whisper"]
```

Reply:
@AathmanT please consider updating the other connectors also with some relevant keywords